src/wavesurfer.js

```diff
@@ -1154,8 +1155,9 @@ export default class WaveSurfer extends util.Observer {
     /**
      * Set the fill color of the waveform behind the cursor.
      *
-     * @param {string} color A CSS color string.
+     * @param {string|object} color A CSS color string, or an array of CSS color strings.
      * @example wavesurfer.setProgressColor('#400');
+     * @example wavesurfer.setWavecolor(['blue','black','yellow']);
```

Comment:
setWaveColor example doesn't belong here.

Reply:
Agreed, removed in latest commit.